MarsQA-1/SpecflowPages/Pages/LanguagesPage.cs

```diff
@@ -0,0 +1,304 @@
+﻿using MarsQA_1.Helpers;
+using NUnit.Framework;
+using OpenQA.Selenium;
+using OpenQA.Selenium.Support.UI;
+using System;
+using System.Collections.ObjectModel;
+using System.Linq;
+using System.Threading;
+
+namespace MarsQA_1.Pages
+{
+    public static class LanguagesPage
+    {
+        private static IWebElement AddNewBtn => Driver.driver.FindElement(By.XPath("//DIV[@class='ui teal button '][text()='Add New']"));
+        private static IWebElement AddLanguageTxt => Driver.driver.FindElement(By.Name("name"));
+        private static IWebElement LanguageLevelDdn => Driver.driver.FindElement(By.Name("level"));
+        private static IWebElement AddBtn => Driver.driver.FindElement(By.XPath("//INPUT[@value='Add']"));
+        private static IWebElement PopUpMsg => Driver.driver.FindElement(By.XPath("//DIV[@class='ns-box-inner']"));
+        private static IWebElement LanguageLbl => Driver.driver.FindElement(By.XPath("//table[@class='ui fixed table']/tbody/tr/td"));
+        private static IWebElement EditIcn => Driver.driver.FindElement(By.XPath("//table[@class='ui fixed table']/tbody/tr/td[3]/span[1]/i"));
+        private static IWebElement UpdateBtn => Driver.driver.FindElement(By.XPath("//input[@class='ui teal button'][@value='Update']"));
+        private static IWebElement DeleteIcn => Driver.driver.FindElement(By.XPath("//i[@class='remove icon']"));
+        private static IWebElement LanguageLevelLbl => Driver.driver.FindElement(By.XPath("//table[@class='ui fixed table']/tbody/tr/td[2]"));
+        private static ReadOnlyCollection<IWebElement> LanguageRecords => Driver.driver.FindElements(By.XPath("//div[@data-tab='first']/div/div[2]/div/table[@class='ui fixed table']/tbody"));
+        private static IWebElement CancelBtn => Driver.driver.FindElement(By.XPath("//INPUT[@class='ui button'][@value='Cancel']"));
+        private static ReadOnlyCollection<IWebElement> AddNewFields => Driver.driver.FindElements(By.XPath("//DIV[@class='fields']"));
```

Comment:
remove static keyword

Reply:
the reason static should not be used like this is that static fields belong to the class there for only one copy is available, if you run your framework in parallel it will not be efficient as it will wait for resources and you will also run into inconsistent test runs

Reply:
see https://www.infoworld.com/article/3045882/static-classes-and-static-class-members-in-c-sharp-explained.html#:~:text=The%20static%20keyword%20in%20the,static%20classes%20and%20static%20members.&text=The%20difference%20between%20a%20static,class%20are%20static%20in%20nature

Reply:
so remove static from class, variable and methods, you will notice that in the common driver it is okay to use static because you will only ever initialise the driver once and then you can use it where as tests some tests will create new objects. Right now its okay but in general you avoid static for classes unless its util or helper classes

Reply:
Removed static keyword every where except util and helper classes.